ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -2231,6 +2231,16 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-gcp-upi-xpn
+- as: e2e-ibmcloud-ipi-p1
+  cron: 0 0 * * *
+  steps:
+    cluster_profile: ibmcloud-qe
+    env:
+      E2E_RUN_TAGS: '@ibmcloud-ipi and @network-ovnkubernetes and not @fips'
```

Comment:
no @network-ovnkubernetes

Reply:
ovnkubernetes is the default network in OCP 4.12 unless openshiftsdn is defined explicitly.